_posts/2020-05-15-Listen-to-what-your-tests-are-mocking.md

```diff
@@ -0,0 +1,151 @@
+---
+title: A Complete Guide to Testing Your Software, Part 2 
+author: Alastair Smith
+category: testing
+created: 1589571968
+tags:
+ - testing
+ - craft
+ - TDD
+ - design
+ - ports and adapters
+---
+
+## <small><i>or, Listen to what your tests are mocking</i></small>
+
+You might remember from [last time](/testing/2020/04/26/complete-guide-testing-your-software-part-1.html) that we
```

Comment:
Link broken (goes to https://github.com/alastairs/codebork.com/blob/master/testing/2020/04/26/complete-guide-testing-your-software-part-1.html)

Reply:
Good catch! This should be fixed in deployment :pray: